### PR TITLE
fix bug where if no "last" cookie, calls to ID5 were not being made

### PR DESCRIPTION
**Pull request by smenzer**
also fixed tests that were missing this case and added a new test specifically for it

### CODE REVIEW

File: src/id5-api.js [hunk 1]
```diff
@@ -29,8 +29,8 @@ ID5.init = function (options) {
     utils.logInfo(`ID5 detected referer is ${referer.referer}`);
 
     const storedResponse = JSON.parse(utils.getCookie(cfg.cookieName));
-    const storedDate = new Date(+utils.getCookie(lastCookieName(cfg)));
-    const refreshNeeded = storedDate.getTime() > 0 && (Date.now() - storedDate.getTime() > cfg.refreshInSeconds * 1000);
```
Comment (smenzer) [inline src/id5-api.js:33]: if there was no cookie found, then `storedDate.getTime()` was `0`, which resulted in this evaluating to `false`

**Review by sgeller — COMMENTED** [review]
Do we want to increase the patch level version for this fix? Otherwise LGTM.

**Comment by smenzer** [comment]
> Do we want to increase the patch level version for this fix? Otherwise LGTM.

yea...i was going to do that as part of release process, but I'll push that now to make sure we don't miss it.

File: src/id5-api.js (hunk 1 above)
Comment (padurgeat) [reply src/id5-api.js:33]: agreed

**Review by padurgeat — APPROVED** [review]

**Pull request merged**